src/index.html

```diff
@@ -7,8 +7,49 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" 
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div>
+        <a href="#home" class="logo"><img src="./images/logo.png" alt="logo"></a>
+      </div>
+      <nav class="nav">
+        <ul class="nav--list">
```

Comment:
```suggestion
        <ul class="nav__list">
```
Так буде правильніше по БЕМ